### DISCUSSION TITLE
How do I go back to the old bar?

### DISCUSSION THREAD

**Discussion by Nexxol** [searxng/searxng#2965] [Q&A] [answered]
This one is just quite frankly, ugly. I liked the previous one more. This one sticks out like a sore thumb.
New
![IMG_6981](https://github.com/searxng/searxng/assets/68960034/86a7b3a5-b81f-4e5e-9d7a-0ac85c481fd6)
Old
![IMG_6982](https://github.com/searxng/searxng/assets/68960034/f8e3a3d6-a908-441a-8f3e-a91ec882503b)

**Comment by dalf** [comment, marked as answer]
The style has not changed, however the HTML and CSS has changed.
Make sure to clean your browser cache.
